docs/UserGuide.md

```diff
@@ -156,16 +156,65 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+ContacX data are saved in the hard disk automatically after any command that changes the data in JSON format at `[JAR file location]/data/contacx.json`.
 
+### Exporting the data: `exportcsv`
+
+Exports the current list as a CSV file that can be imported through ContacX or other contact software such as Google/Outlook
+
+Format: `exportcsv EXPORTTYPE`
+* Exports CSV as per the specified format.
+* If `EXPORTTYPE` is left empty, it will default to `EXPORTTYPE = 1` and export in custom ContacX format
+* `EXPORTTYPE` options are
+   * `EXPORTTYPE=1` for ContacX format
+   * `EXPORTTYPE=2` for Google Contacts format
+   * `EXPORTTYPE=3` for Microsoft Outlook format
+* File will be saved on the directory `[JAR file location]/data/addressbook.csv`
+
+Examples:
+* `exportcsv 2`: Exports the current address book as a CSV file ready to be uploaded onto Google Contacts 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+ContacX data are saved as a JSON file `[JAR file location]/data/contacx.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Spelling issue. It is also present in multiple other locations.
```suggestion
ContaX data are saved as a JSON file `[JAR file location]/data/contax.json`. Advanced users are welcome to update data directly by editing that data file.
```

Reply:
thanks for catching will stop renaming stuff wrongly